src/services/fileStorage/proxy-service.js

```diff
@@ -33,9 +34,21 @@ const logger = require('../../logger');
 
 const FILE_PREVIEW_SERVICE_URI = process.env.FILE_PREVIEW_SERVICE_URI || 'http://localhost:3000/filepreview';
 const FILE_PREVIEW_CALLBACK_URI = process.env.FILE_PREVIEW_CALLBACK_URI
-|| 'http://localhost:3030/fileStorage/thumbnail/';
+	|| 'http://localhost:3030/fileStorage/thumbnail/';
```

Comment:
NO constants here, move them all to a configuration file, at least: new ticket

Reply:
https://ticketsystem.schul-cloud.org/browse/SC-2381